railties/lib/rails/rack/logger.rb

```diff
@@ -79,6 +82,12 @@ def development?
       def logger
         Rails.logger
       end
+
+      def show_extra_line_in_development_for_non_assets_requests(path)
+        if development? && path !~ /^\/assets/
```

Comment:
Does this work safely to handle public assets, assets from engines, vendored assets, etc?

btw, this won't work with http://guides.rubyonrails.org/asset_pipeline.html#changing-the-assets-path